src/ontology/mondo-edit.obo

```diff
@@ -62228,10 +62226,11 @@ xref: ICD9:307.49 {source="MONDO:relatedTo", source="MONDO:i2s"}
 xref: MEDGEN:167920 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: MESH:D012893 {source="MONDO:equivalentTo"}
 xref: SCTID:39898005 {source="MONDO:equivalentTo"}
-xref: UMLS:C0877792 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:167920"}
+xref: UMLS:C4042891 {source="https://orcid.org/0000-0002-3458-4839"}
```

Comment:
UMLS mappings are maintained by MEDGEN. So this x-ref will be automatically updated back to what it was during the next externally managed content update. If we truly believe that this UMLS mapping needs to be updated, we should let MedGen know, and they should de the change on their side.

Reply:
How do we let MedGen know about the change in the xref?

Reply:
You need to let MedGen know by getting in contact with their team. Our contact is Megan, who submitted the issue. Therefore the easiest way is to comment on the issue (and tag her), explain what your investigation found and what you did, and suggest that the UMLS x-ref should be different. 

In general, she (and users in general) will most likely not see any of the comments in a PR. Therefore everything that is in the description of this PR should be shared in the main issue so she can see it and comment.